modules/flight_interface/flight_interface_worker.py

```diff
@@ -38,4 +41,13 @@ def flight_interface_worker(
         if not result:
```

Comment:
pass instead of continue, if we don't get an printer
odometry but do get a command we do want to send the command